docs/application/native/tutorials/details/native-removed-history.md

```diff
@@ -0,0 +1,39 @@
+# Removed functions and enumerations of Tizen Native API
+
+This page describes the functions and enumerations that were removed since Tizen 3.0.
+
+If you want to know the lifecycle of Tizen Native API, you need to read the contents of [API Versioning and Deprecation Policy of the Tizen Platform](deprecation-policy.md) first.
+
+## Detailed information regarding removed functions and enumerations

```

Comment:
I feel this heading is not required.
Instead add a text like this:
The following table provides detailed information about the removed functions and enumerations: